postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -358,29 +381,60 @@ def _run_explain(self, dbname, statement, obfuscated_statement):
                 return None
             return result[0][0]
 
+    @staticmethod
+    def _translate_setup_state_err_to_reason(state, err):
+        return {
+            DBExplainSetupState.failed_connect: {
+                'code': FailedExplainReason.database_error.value,
+                'reason': '{}'.format(repr(err)),
```

Comment:
I think there is a risk in leaking bind parameters if you just print the error exactly here. The error class should give sufficient indication of what went wrong.